docs/target-compatibility.md

```diff
@@ -56,6 +57,14 @@ tex.GetDimensions is the GetDimensions method on 'texture' objects. This is not
 
 SM6.5 Wave Intrinsics are supported, but requires a downstream DXC compiler that supports SM6.5. As it stands the DXC shipping with windows does not. 
 
+## WaveShuffle
+
+WaveShuffle is an intrinsic added to the Slang stdlibrary to expose the glsl `subgroupShuffle` intrinsics and allow loosened requirements on laneId. 
+
+In `HLSL` requires the `laneId` must be 'dynamically uniform' across the wave. WaveShuffle has the same functionality but relaxes this restriction. 
+
+`WaveReadLaneAt` most obviously maps to `subgroupBroadcast` in GLSL. This has the extra restriction the index must be compile time consts. With SPIR-V 1.4 it is allowed to be 'dynamically uniform', but doesn't work on current glslang.
```

Comment:
The comment about SPIR-V 1.4 here seems inaccurate w.r.t. the extension spec, as covered in the review of the previous PR. We should double check the facts here before checking this in.